common/views/components/OutboundLinkTracker/OutboundLinkTracker.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+
+import React, { useEffect } from 'react';
+import ReactGA from 'react-ga';
+import type { Node } from 'react';
+
+type Props = {|
+  children: Node,
+|};
+
+const OutboundLinkTracker = ({ children }: Props) => {
+  function getDomain(url: string): string {
```

Comment:
Do these functions need to be in the component?

Reply:
no – will move to a `domain` util